resource-managers/kubernetes/core/src/main/scala/org/apache/spark/deploy/k8s/Constants.scala

```diff
@@ -63,6 +63,9 @@ object Constants {
   val UI_PORT_NAME = "spark-ui"
 
   // Environment Variables
+  val ENV_DRIVER_ATTRIBUTE_APP_ID = "SPARK_DRIVER_ATTRIBUTE_APP_ID"
+  val ENV_DRIVER_ATTRIBUTE_KUBERNETES_NAMESPACE = "SPARK_DRIVER_ATTRIBUTE_KUBERNETES_NAMESPACE"
```

Comment:
Just a question: we need this K8s namespace attribute additionally only for driver logs?

Reply:
yes